src/lib/Storage/ContentDataSource.php

```diff
@@ -22,59 +23,69 @@
 use Ibexa\Personalization\Value\Storage\Item;
 use Ibexa\Personalization\Value\Storage\ItemList;
 use Ibexa\Personalization\Value\Storage\ItemType;
+use Psr\Log\LoggerInterface;
 
 final class ContentDataSource implements DataSourceInterface
 {
     private SearchService $searchService;
-
     private ContentService $contentService;
-
     private QueryType $queryType;
-
```

Comment:
Watch out, because new code style will require one empty line between class properties.

There is no PR yet.

Reply:
https://github.com/ezsystems/ezplatform-code-style/pull/17

Reply:
Fixed: https://github.com/ezsystems/ezrecommendation-client/pull/74/commits/b706e3f3b985de6bc1d1e919bcc737d117909287